src/axom/sidre/spio/IOManager.cpp

```diff
@@ -793,14 +796,18 @@ std::string IOManager::getFileNameForRank(const std::string& file_pattern,
 
   //If the root file was given as a path,
   //find the directory and add it to file_name
-  std::string curr;
-  std::string root_dir;
-  std::string slash = "/";
-  conduit::utils::rsplit_string(root_name, slash, curr, root_dir);
+  std::string delimiter = "/";
+  axom::Path root_path(root_name);
+  std::string root_dir = root_path.dirName();
+
+  if(!root_name.empty() && root_name[0] == delimiter[0])
```

Comment:
Suggestion: It might be cleaner to replace this with `axom::utilities::string::startsWith(root, delimiter[0])`

See: https://github.com/LLNL/axom/blob/7d595645e7e04e1f88c0f5b5cb265e7328eb0859/src/axom/core/utilities/StringUtilities.hpp#L43-L64